database/tables/batched_migrations.pg

```diff
@@ -0,0 +1,14 @@
+columns
+    current: bigint not null
+    finished_at: timestamp with time zone
+    id: bigint not null default nextval('batched_migrations_id_seq'::regclass)
+    max: bigint not null
+    min: bigint not null
+    name: text
+    project: text default 'prairielearn'::text
+    started_at: timestamp with time zone default CURRENT_TIMESTAMP
+    timestamp: text
```

Comment:
`timestamp` is something like `20230411211407` that we pull directly from the migration filename. In the case of batched migrations, it's really more of a unique identifier for a specific migration file; we don't actually rely on it for ordering (although we did in intermediate implementations), since ordering is determined by the order of the regular migrations that actually enqueue the batched migrations. Maybe for batched migrations specifically, we can get away without using a timestamp and just rely on the filename (minus extension) as a unique identifier? That might be preferable so that folks don't wrongly assume that the `timestamp` portion of the filename will influence ordering.

Reply:
I would prefer to just use the "timestamp-like" prefix of the filename as the identifier, to allow us to rename the rest of the filename without breaking everything (I think we've done this in the past for regular migrations).

Reply:
Maybe we could rename the `timestamp` column to makes that more clear?

Reply:
This nomenclature is copied directly from the existing `migrations` table. I'd rather keep consistent terminology for the same concept.